x-pack/plugins/synthetics/server/synthetics_service/formatters/public_formatters/format_configs.ts

```diff
@@ -7,17 +7,17 @@
 
 import { isEmpty, isNil, omitBy } from 'lodash';
 import { Logger } from '@kbn/logging';
-import { PARAMS_KEYS_TO_SKIP } from '../../../common/formatters/format_synthetics_policy';
-import { replaceStringWithParams } from '../../../common/formatters/formatting_utils';
 import {
   BrowserFields,
   ConfigKey,
+  HeartbeatConfig,
   MonitorFields,
   SyntheticsMonitor,
-  HeartbeatConfig,
   TLSFields,
-} from '../../../common/runtime_types';
-import { formatters } from '.';
+} from '../../../../common/runtime_types';
+import { publicFormatters } from '.';
+import { PARAMS_KEYS_TO_SKIP } from '../private_formatters/format_synthetics_policy';
```

Comment:
Why is the private being imported into the public? Perhaps we should put this somewhere higher up in common to make it clear it's relevant to both? Same with the below import.